src/store/actions.js

```diff
@@ -105,16 +104,18 @@ export default {
       })
   },
   retrieveNegotiationById({ state, commit }, { negotiationId }) {
-    return axios.get(`${NEGOTIATION_PATH}/${negotiationId}`, getBearerHeaders(state.oidc.access_token))
+    return axios.get(`${NEGOTIATION_PATH}/${negotiationId}`, {headers : getBearerHeaders(state.oidc.access_token)})
       .then((response) => {
         return response.data
       })
       .catch(() => {
         commit("setNotification", "Error getting request data from server")
       })
   },
-  retrievePostsByNegotiationId({ state, commit }, { negotiationId }) {
-    return axios.get(`${NEGOTIATION_PATH}/${negotiationId}/posts`, getBearerHeaders(state.oidc.access_token))
+  retrievePostsByNegotiationId({ state, commit }, { negotiationId, type, resourceId }) {
+    let url = resourceId ? `${NEGOTIATION_PATH}/${negotiationId}/posts?` : `${NEGOTIATION_PATH}/${negotiationId}/posts?type=${type}`
```

Comment:
you should remove the `?type` part of the url and keep the url  `${NEGOTIATION_PATH}/${negotiationId}/posts